src/Sarif/VersionOne/SarifUtilitiesVersionOne.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.CodeAnalysis.Sarif.VersionOne;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public static class SarifUtilitiesVersionOne
+    {
+        private const string V1_0_0 = "1.0.0";
+        private const string V1_0_0_BETA_5 = "1.0.0-beta.5";
+
+        public static string ConvertToText(this SarifVersionVersionOne sarifVersion)
+        {
+            switch (sarifVersion)
+            {
+                case SarifVersionVersionOne.OneZeroZeroBetaFive: { return V1_0_0_BETA_5; }
```

Comment:
I would throw `ArgumentException` if you are asked to convert anything except `V1_0_0`.

Reply:
Done